SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/commands/Test.swift

```diff
@@ -24,9 +24,8 @@ struct Test: ParsableCommand, BuildCommand {
   var arguments: BuildArguments
 
   func run() throws {
-    try buildExample(exampleName: "ExamplePlugin")
```

Comment:
Curious, and unrelated to the PR: why remove the `buildExample` step? Do we build in another command now? (Just learning about the scripts, and will read more — very likely that I'll stumble at the answer on my own in 5 minutes)

Reply:
Oh, we needed to build the example for the lit test, which I removed in https://github.com/apple/swift-syntax/pull/2160. I just forgot to remove `buildExample` in that PR.